lesson/outline/learning-standards.md

```diff
@@ -0,0 +1,411 @@
+## Example: What is Jest?
+
+### Learning Standard Text
+
+From the [documentation](https://jestjs.io/), "Jest is a JavaScript testing framework designed to ensure correctness of any JavaScript codebase. It allows you to write tests with an approachable, familiar and feature-rich API that gives you results quickly."
+
+### Tags
+
+JavaScript, Testing, Jest
+
+### Additional Notes / Resources Used
+
+Sources:
+* https://jestjs.io/
+
+<hr>
+
+## Installing Jest
+
+### Learning Standard Text
+
+If you are using `create-react-app` to initialize your React project, Jest will already be included. 
+
+To manually install Jest with `npm`, use the following command:
+
+```sh
+npm install --save-dev jest
+```
+
+Though not required, the `--save-dev` flag will add this library as a development dependency rather than a production dependency. You can see this in the `package.json` file:
+
+```js
+"devDependencies": {
+  "jest": "^26.6.3"
+},
+```
+
+### Tags
+
+JavaScript, Testing, Jest
+
+### Additional Notes / Resources Used
+
+Sources:
+* https://jestjs.io/docs/getting-started
+
+<hr>
+
+## Running Jest Tests
+
+### Learning Standard Text
+
+To run a test using Jest, first add the following to your `package.json` file:
+
+```js
+{
+  "scripts": {
+    "test": "jest"
+  }
+}
+```
+
+Then, you can run `npm test` from your terminal. 
+
+Jest will automatically look for files that have the `.test.js` or `.spec.js` suffix (as well as `.js` files in a `__test__` directory).
+
+
+### Tags
+
+JavaScript, Testing, Jest
+
+### Additional Notes / Resources Used
+
+Sources:
+* https://jestjs.io/docs/getting-started
+
+<hr>
+
+## The `test()` function
+
+### Learning Standard Text
+
+Every Jest test begins with the `test()` function, which accepts two arguments:
```

Comment:
Should we call out that some codebases use `it` instead? https://jestjs.io/docs/api#testname-fn-timeout